pkg/webhook/v1alpha1/clusterresourcequota_webhook.go

```diff
@@ -164,7 +144,8 @@ func (h *ClusterResourceQuotaWebhook) Handle(c *gin.Context) {
 	c.JSON(http.StatusOK, admissionReview)
 }
 
-func (h *ClusterResourceQuotaWebhook) validateCreate(
+// validateOperation is a shared helper for create/update validation
+func (h *ClusterResourceQuotaWebhook) validateOperation(
```

Comment:
Does this get called whenever an object is updated? Won't this cause unnecessary noise? Shouldn't this be calculated only on creation and deletion? I'm asking because I may have missed something.